docs/js/class_p.js

```diff
@@ -8659,26 +8651,6 @@ class Puzzle {
         }
     }
 
-    mouse_numberS(x, y, num, submode) {

```

Comment:
Is there a reason, you removed this piece of code? The only purpose of this code was to improve the experience with certain types of puzzles that involve both big digits and corner digits as part of solution. E.g. Tight Fit Sudoku